api/udpa/data/orca/v1/orca_load_report.proto

```diff
@@ -22,10 +22,13 @@ message OrcaLoadReport {
   // during the request.
   double mem_utilization = 2 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
 
+  // Total QPS being served by an endpoint.
```

Comment:
@vishalpowar I'm wondering how this works for endpoints supporting multiple services and inline vs. OOB.

With inline, it would make sense to report the QPS for the current service. But, for OOB, the agent on the node might not know about the distinction between different services. How would it even know the total QPS unless there is some additional per-request reporting to the agent?

Reply:
For multiple services on an endpoint, would the utilization be reported per service or for the whole endpoint ?

The QPS is used in conjunction with the utilization of the backend, to determine the impact of RPC on the server (to make better LB decision).
We should ideally have the reporting (utilization and QPS) at the same granularity for both OOB and inline reporting.

Reply:
I think your question is the same as my one :) I'm wondering if you can look at how the gRPC-LB client-side algorithm you are planning to use expects QPS to be reported based on some of our historical use of this metric in client-side LB?

Reply:
I suspect that utilization will always be for the backend server process as a whole.  If there are multiple services being served on the same port (or even on multiple ports in the same process), I think they will all be lumped together in terms of utilization.  This is because utilization is ultimately a percentage of some total, and most servers don't have a good way to internally partition the resources within a given process to different services or ports.

I think the utilization data would wind up being the same, regardless of whether it is sent per-RPC or OOB.

@vishalpowar, I suggest double-checking what I just said with Jonathon to make sure I'm not lying. :)

Reply:
I checked, it is the qps for all the services running on the backend.